tests/js/server/aql/aql-skipping-cluster.js

```diff
@@ -70,6 +70,16 @@ function aqlSkippingClusterTestsuite () {
         assertEqual(i, n);
       }
     },
+
+    testSkipWithFullCount: function () {
```

Comment:
@mpoeter Could you add a short comment that this is a regression test for BTS-442?

Reply:
Done